src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -79,7 +79,9 @@
                   <p class="people-roles-title mb-1" :class="{ 'removed': wasRemoved(orgPerson)}">
                     {{ formatName(orgPerson) }}
                   </p>
-                  <p class="info-text mb-1 people-roles-email" :class="{ 'removed': wasRemoved(orgPerson)}">
+                  <p v-if="!(isLimitedConversionRestorationFiling || isLimitedExtendRestorationFiling)"
+                    class="info-text mb-1 people-roles-email"
+                    :class="{ 'removed': wasRemoved(orgPerson)}">
                     {{ orgPerson.officer.email }}
```

Comment:
Is there a way to decouple this?  If `ListPeopleAndRoles` is a common component, it would be better not to add logic that's specific to a restoration filing.  It's not ideal, but perhaps we could add a getter called something like, `showOfficerEmail` and have the business logic in the getter instead?

Reply:
I agree. I was on the fence while adding this as well.

Reply:
If possible, this logic should be abstracted (ie, not so much whether this is not a resto filing, but whether this is displayed or not), and then controlled by a resource file property.

Reply:
Also, I highly recommend the use of positive logic (ie, in which scenarios do you want to display this block?) instead of negative logic (ie, for which filings does this NOT apply). The reasons are:
- easier to understand
- easier to find what _should_ be displayed instead of what _shouldn't_ be displayed -- if we add a filing type then curently this would automatically be displayed; it would be better to not see it and then need to add a condition to show it

Reply:
LGTM. Jonathan, your conversation to resolve when you are satisfied.